src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -3434,6 +3445,62 @@ class XmlFileFormatTypeParameters(BaseSegment):
     )
 
 
+class AlterPipeSegment(BaseSegment):
+    """A snowflake `Alter PIPE` statement.
+
+    https://docs.snowflake.com/en/sql-reference/sql/alter-pipe.html
+    """
+
+    type = "alter_pipe_segment"
+    match_grammar = Sequence(
+        "ALTER",
+        "PIPE",
+        Ref("IfExistsGrammar", optional=True),
+        Ref("ObjectReferenceSegment"),
+        OneOf(
+            Sequence(
+                "SET",
+                AnyNumberOf(
+                    Sequence(
+                        "PIPE_EXECUTION_PAUSED",
+                        Ref("EqualsSegment"),
+                        Ref("BooleanLiteralGrammar"),
+                    ),
+                    Ref("CommentEqualsClauseSegment"),
+                ),
+            ),
+            Sequence(
+                "UNSET",
+                OneOf("PIPE_EXECUTION_PAUSED", "COMMENT"),
+            ),
+            Sequence(
+                "SET",
+                Ref("TagEqualsSegment"),
+            ),
+            Sequence(
+                "UNSET",
+                Sequence("TAG", Delimited(Ref("NakedIdentifierSegment"))),
+            ),
+            Sequence(
+                "REFRESH",
+                Sequence(
+                    "PREFIX",
+                    Ref("EqualsSegment"),
+                    Ref("PrefixPath"),
+                    optional=True,
+                ),
+                Sequence(
+                    "MODIFIED_AFTER",
+                    Ref("EqualsSegment"),
+                    Ref("TimeStampISO8601"),
```

Comment:
Since quotes are required, do we need a specific regex for the contents of the string?  For example, could we use `single_quote` instead? Also, are double quotes supported (even though not explicitly mentioned in the documentation)?

Reply:
according to the doc: 
```
MODIFIED_AFTER = 'start_time'
Timestamp (in ISO-8601 format) of the oldest data files to copy into the Snowpipe ingest queue based on the LAST_MODIFIED date (i.e. date when a file was staged).
```
So I used the regex in order to check that it's not just the any number of symbol
If you think it's better not to use regex there, I can make changes

Reply:
Let's remove it. In the actual database, the format of the string would most likely be checked at execution time, not lex or parse time, so SQLFluff should do the same.

Reply:
Done!